### DISCUSSION TITLE
Question: Do we still need this if we have ts-context-commentstring?

### DISCUSSION THREAD

**Discussion by rubiin** [folke/ts-comments.nvim#29] [Q&A] [answered]
been using [nvim-ts-context-commentstring](https://github.com/JoosepAlviste/nvim-ts-context-commentstring/tree/main) with comment.nvim

**Comment by folke** [comment, marked as answer]
ts-comments is for native neovim comments only, so with comments.nvim you'll need ts-context-commentstring
